chart/templates/_tplvalues.tpl

```diff
@@ -0,0 +1,13 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
What's the need for this special function? We can simply have the label values as is, no?

Reply:
The function is now removed